media/js/mozorg/manifesto.js

```diff
@@ -57,16 +52,30 @@ $(function() {
                         .text(window.trans('principle-nav-prev')).appendTo($nav);
 
                     $nav.on('click', 'button', function () {
-                        nav_modal($(this).hasClass('prev') ? -1 : 1);
-                    });
 
-                    window.dataLayer.push({
-                        event: 'manifesto-interaction',
-                        browserAction: 'modal open',
-                        section: section_id.match(/\d+/)[0]
+                        var quoteText = $('.principle.overlay-contents h3').text();
+                        // Track arrow clicks on Manifesto quote popup
+
+                        window.dataLayer.push({
+                            event: 'manifesto-quote-click-arrow',
+                            direction: ($(this).attr('class') === 'prev') ? 'previous' : 'next',
+                            quote: quoteText.slice(3),
+                            quoteNumber: Number(quoteText.slice(0, 2))
+                        });
+
+                        nav_modal($(this).hasClass('prev') ? -1 : 1);
```

Comment:
Since `$(this)` is used twice, lets store that reference for better performance, like `var $this = $(this);`